docs/Other SDL Features/Creating an App Service/index.md

```diff
@@ -408,7 +408,7 @@ sdlManager.fileManager.upload(file: artwork) { [weak self] (success, bytesAvaila
 
 @![android,javaSE,javaEE]
 ```java
-final SdlArtwork navInstructionArt = new SdlArtwork("turn", FileType.GRAPHIC_PNG, R.drawable.turn, true);
+final SdlArtwork navInstructionArt = new SdlArtwork("turn", FileType.GRAPHIC_PNG, <#image#>, true);
```

Comment:
Should this be made more clear which constructor is used or what data type actually goes here? such as the following
<#image_resourceId#>
<#image_uri#>
<#image_byteArray#>